src/packages/cli/.gitignore

```diff
@@ -11,7 +11,6 @@ download-build
 runtime
 examples
 !sqlite.test.db
-!src/__tests__/fixtures/studio-test-project/dev.db
```

Comment:
This didn't seem to be working so instead I just renamed to `db.sqlite`. Thoughts?

Reply:
all our sqlite db are name `name.db` so maybe don't rename only one

Reply:
Cool I'll stick to that convention!